.github/workflows/wheel_macOS.yml

```diff
@@ -0,0 +1,66 @@
+name: wheel - macOS
+
+on:
+  push:
+    tags:
+      - '*'
+jobs:
+  wheel_macos:
+    name: macOS wheel
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-latest]
+        python-version: [3.7, 3.8]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: check C++ compiler and Python
+        run: |
+          clang --version
+          python --version
+      - name: Install dependencies
+        run: |
+          python -m pip install --no-cache-dir --upgrade pip
+          python -m pip install --no-cache-dir "poetry<2,>=1.0"
+          python -m pip install --no-cache-dir "mypy-protobuf~=1.0"
+      - name: Build libraries
+        run: |
+          make
+      - name: Build wheel
+        run: |
+          python -m poetry build -v
+      - name: Fix wheel
+        run: |
+          python -m pip install wheel
+          python -m pip install delocate
+          delocate-wheel -v dist/*.whl -w ./wheelhouse
+          delocate-addplat -x 10_9_intel -x 10_10_intel .wheelhouse/*.whl
```

Comment:
I'd like to double check that these are the delocate command we currently run in httpstan-wheels. I think it's important that we first reproduce what we are doing currently (since we know it works). Once it's working we can experiment with changes to the procedure.

Reply:
I don't believe we use delocate-addplat in httpstan-wheels right now. Did you have a reason for adding it?

Reply:
I believe `delocate-wheel` is the only command which is required.